chain/network/src/config.rs

```diff
@@ -318,6 +320,7 @@ impl NetworkConfig {
             outbound_disabled: false,
             archive,
             accounts_data_broadcast_rate_limit: rate::Limit { qps: 0.1, burst: 1 },
+            snapshot_hosts_broadcast_rate_limit: rate::Limit { qps: 0.1, burst: 1 },
```

Comment:
question for my own education. How's this different from the one on line 392?

Reply:
Line 392 is part of a function which generates a config for local testing purposes:
https://github.com/near/nearcore/blob/07dd3136d6d8bfbd6d1ce0aec00dc10d96520dee/chain/network/src/config.rs#L348-L406